resources/tentative/newssite/news-next/src/components/toggle/Toggle.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+
+export default function Toggle({ label, onChange, checked }) {
+    const [isSelected, setIsSelected] = useState(false);
+
+    useEffect(() => {
+        setIsSelected(checked);
+    }, [checked]);
```

Comment:
It's not clear to me why we need a local state instead of using directly the prop `checked`.

Reply:
The toggle component uses a "checked" prop, which will let us assign the initial state, or change the toggle from a parent component.